client/post-editor/post-editor.jsx

```diff
@@ -554,7 +606,9 @@ var PostEditor = React.createClass( {
 				}
 			}.bind( this );
 		}
+		this.props.autosave( callback );
```

Comment:
Well, the callback should die, obviously
I just passed it 'in case', because I have no idea how will we implement these actions.
I think it will be useful only while figuring out what is the current state, then it should be removed.